integration_tests/ctesque/src/sharedTest/java/android/app/ActivityTest.java

```diff
@@ -77,4 +78,16 @@ public void whenNotSetOnActivityInManifest_activityGetsThemeFromApplicationInMan
           });
     }
   }
+
+  @Test
+  public void launchActivity_abstractActivity_throwsRuntimeException() {
+    boolean catchException = false;
+    try (ActivityScenario<AbstractTestActivity> scenario =
+        ActivityScenario.launch(AbstractTestActivity.class)) {
+      assertThat(scenario).isNull();
+    } catch (RuntimeException exception) {
+      catchException = true;
+    }
+    assertThat(catchException).isTrue();
+  }
```

Comment:
Wouldn't it be enough to do the followng:
```suggestion
  @Test(expected = RuntimeException.class)
  public void launchActivity_abstractActivity_throwsRuntimeException() {
    try (ActivityScenario<AbstractTestActivity> scenario =
                 ActivityScenario.launch(AbstractTestActivity.class)) {
      assertThat(scenario).isNull();
    }
  }
```